### PR TITLE
Add serviceable attribute to projects.

### PR DESCRIPTION
**Pull request by NTaylorMullen**

### CODE REVIEW

File: src/EntityFramework/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyMetadata("Serviceable", "True")]
```
Comment (bricelam) [inline src/EntityFramework/Properties/AssemblyInfo.cs]: There isn't a project in this directory.

File: src/Shared/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyMetadata("Serviceable", "True")]
```
Comment (bricelam) [inline src/Shared/Properties/AssemblyInfo.cs]: There isn't a project in this directory either

File: src/EntityFramework/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (NTaylorMullen) [inline src/EntityFramework/Properties/AssemblyInfo.cs]: :+1: scripting ftw

File: src/Shared/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (NTaylorMullen) [inline src/Shared/Properties/AssemblyInfo.cs]: :+1: scripting ftw

**Comment by NTaylorMullen** [comment]
Updated.

**Comment by lukewaters** [comment]
:shipit:

**Pull request merged**